discoveries/projected-changes-WUS-snow.discoveries.mdx

```diff
@@ -0,0 +1,142 @@
+  </Chapter>
+  <Chapter
+    center={[-114.80,47.40]}
+    zoom={6.5}
+    datasetId='snow-projections-diff'
+    layerId='snow-projections-diff-scenario-245'
+    datetime='2077-05-01'
+  >
+    ## Northern Rocky Mountains
+    Although at higher elevations and colder temperatures than the Washington Cascades, average end-of-century snowpack is expected to reduce by 48%, relative to the historical period (1995 - 2014). The largest projected decreases occur in Northern Idaho and Northwest Montana. Some portions of Northern Montana near Glacier National Park could have temperatures that are sufficiently cold enough to sustain snowfall with future warming and increases in precipitation, resulting in small (0-15%, 6% on average) increases in May 1 snow volume in those regions.   
+  </Chapter>
+  <Chapter
+    center={[-108.90,40.70]}
+    zoom={5.5}
+    datasetId='snow-projections-diff'
+    layerId='snow-projections-diff-scenario-245'
+    datetime='2077-05-01'
+  >
+    ## Central and Southern Rocky Mountains
+    Although end-of-century winter temperatures in this region are projected to increase by approximately 3 degrees Celsius (emissions scenario SSP2-4.5), winter temperatures may be sufficiently cold enough to sustain snowfall at high elevations. This could result in high-elevation increases in May 1 snowpack, particularly in the cold Wyoming Rockies. However, projections often disagree on whether high-elevation winter snow will increase or decrease, and high elevation increases are not large enough to offset lower-elevation decreases in snow volume.   
+  </Chapter>
+</ScrollytellingBlock> 
+<Block>
+   <Figure>
+     <Image 
+       src={new URL("./habitat_spaghettiPlots.jpg", import.meta.url).href}
+       alt="changes to thresholded habitat across elevation bands" 
+      />
+   </Figure>
+   <Prose>
+   The snow projections developed here can be used for multiple applications. For example, wolverine habitat assessments performed by the US Fish and Wildlife Service suggested that snowy regions were more likely to support denning if mid-spring (e.g., May 15) snow depth exceeded 0.5m. Using these projections, we could identify the percent-change to the amount of area that had snow depth exceeding the spring snow depth threshold for different elevation bands. The plots shown here show the changes across elevation bands for the four future periods (emissions scenario SSP2-4.5), for three domains, accounting for both the ensemble mean (line) and spread (shaded) across 23 different climate models. Note that although some of the colder domains like the Colorado Rockies and Montana Rockies had high-elevation increases to end-of-century projected snow, the prevelence of area exceeding this habitat threshold only decreased for all elevation bands in the last simulation period (2076 - 2095).
+   </Prose>
+</Block>  
+
+<Block type='wide'>
```

Comment:
Consider whether to keep the `wide` block for such little text, as it shows up in a partial column looking broken.

Reply:
@jupflug Should this block be wide (last in the page)

<img width="1220" alt="image" src="https://user-images.githubusercontent.com/1090606/231464730-c9d70482-077c-4c77-853e-d643aa42255c.png">